File: C#/begginer/1064.cs
using System; 

class URI {

  static void Main(string[] args) { 
    double average = 0;
    int count = 0;
    for(int i = 0; i < 6; i++) {
      double input = double.Parse(Console.ReadLine());
      if (input > 0) {
        count++;
        average += input;
      }
    }
    Console.WriteLine($"{count} valores positivos");
    Console.WriteLine($"{average / count:F1}");
  }

}

File: Java/begginer/1064.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
        Scanner scanner = new Scanner(System.in);
        double sum = 0;
        int count = 0;
        
        for(int i = 0; i < 6; i++) {
            double input = scanner.nextDouble();
            if(input > 0) { sum += input; count++; }
        }
        
        System.out.printf("%d valores positivos\n", count);
        System.out.printf("%.1f\n", sum / count);
    }
 
}
